github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-periodics.yaml

```diff
@@ -97,21 +97,16 @@ periodics:
       secret:
         secretName: gcs
     containers:
-    - image: quay.io/kubevirt/builder:2108311443-370e711b8
+    - image: quay.io/kubevirtci/pr-creator:v20210902-265ef38
```

Comment:
I get an error with the given tag, this one works
```suggestion
    - image: quay.io/kubevirtci/pr-creator:v20210901-354d148
```

Reply:
Thanks, great catch. Must have been a locally created image which I didn't push... PTAL!